presto-iceberg/src/main/java/com/facebook/presto/iceberg/equalitydeletes/EqualityDeletesSplitSource.java

```diff
@@ -107,6 +108,8 @@ private IcebergSplit splitFromDeleteFile(DeleteFile deleteFile)
                 deleteFile.format(),
                 ImmutableList.of(),
                 getPartitionKeys(specById.get(deleteFile.specId()), deleteFile.partition()),
+                PartitionSpecParser.toJson(PartitionSpec.unpartitioned()),
+                Optional.empty(),
```

Comment:
Is that ok? As `partitionSpec` and `partitionData` are all available, may be we can set them properly.